### PR TITLE
Standardize Java packages

### LINKED ISSUE #185: Repo cleanup - remove scripts, unused test data
Remove some repo files that aren't part of the normal test process or are user settings:

- ~.code-workspace files~ (these are needed)
- /events-0.0.3
- /postgres
- /NodeCollector (unless this is important)

Cleanup .gitignore with related files grouped (such as /config/...)

### LINKED ISSUE #339: Standardize package naming
**Problem**
The project uses non-standard camelCase for some package names. It also doesn't nest some packages under a mirror node specific package to avoid namespace collisions.

**Solution**
- Move all package under `com.hedera.mirror`
- Rename all packages to be lowercase
- Move classes from `databaseUtilities`, `filedelimiters` and `utilities` into `com.hedera.mirror.util`
- Move `recordFileLogger` classes into `com.hedera.mirror.parser.record`
- Move `platform` classes into `com.hedera.mirror.parser.event`
- Adjust all test packages similarly

**Alternatives**
- Leave as is
- Fix as we go

**Additional Context**
#161

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
- Moves all classes under `com.hedera.mirror`
- Consolidates util classes into `com.hedera.mirror.util`
- Removes some unused files

**Which issue(s) this PR fixes**:
Fixes #185
Fixes #339 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: src/main/java/com/hedera/mirror/parser/event/EventStreamFileParser.java [hunk 1]
```diff
@@ -44,14 +44,13 @@
 import org.apache.commons.codec.binary.Hex;
 import org.springframework.scheduling.annotation.Scheduled;
 
-import com.hedera.databaseUtilities.DatabaseUtilities;
-import com.hedera.filedelimiters.FileDelimiter;
+import com.hedera.mirror.util.DatabaseUtilities;
+import com.hedera.mirror.util.FileDelimiter;
```
Comment (mike-burrage-hedera) [inline src/main/java/com/hedera/mirror/parser/event/EventStreamFileParser.java]: sorting-wise should these be 3 lines lower?

File: src/main/java/com/hedera/mirror/parser/balance/AccountBalancesFileLoader.java [hunk 2]
```diff
@@ -36,7 +36,7 @@
 import lombok.Getter;
 import lombok.extern.log4j.Log4j2;
 
-import com.hedera.databaseUtilities.DatabaseUtilities;
+import com.hedera.mirror.util.DatabaseUtilities;
```
Comment (mike-burrage-hedera) [inline src/main/java/com/hedera/mirror/parser/balance/AccountBalancesFileLoader.java]: import sorting?

File: src/main/java/com/hedera/mirror/parser/record/RecordFileLogger.java [hunk 3]
```diff
@@ -49,10 +49,9 @@
 
 import lombok.extern.log4j.Log4j2;
 
-import com.hedera.databaseUtilities.DatabaseUtilities;
+import com.hedera.mirror.util.DatabaseUtilities;
```
Comment (mike-burrage-hedera) [inline src/main/java/com/hedera/mirror/parser/record/RecordFileLogger.java]: import sorting?

File: src/main/java/com/hedera/mirror/parser/record/RecordFileParser.java [hunk 4]
```diff
@@ -39,14 +39,13 @@
 import org.apache.commons.codec.binary.Hex;
 import org.springframework.scheduling.annotation.Scheduled;
 
-import com.hedera.filedelimiters.FileDelimiter;
+import com.hedera.mirror.util.FileDelimiter;
```
Comment (mike-burrage-hedera) [inline src/main/java/com/hedera/mirror/parser/record/RecordFileParser.java]: import sorting?

File: src/main/java/com/hedera/mirror/parser/balance/AccountBalancesFileLoader.java (hunk 2 above)
Comment (steven-sheehy) [reply src/main/java/com/hedera/mirror/parser/balance/AccountBalancesFileLoader.java]: Done

File: src/main/java/com/hedera/mirror/parser/event/EventStreamFileParser.java (hunk 1 above)
Comment (steven-sheehy) [reply src/main/java/com/hedera/mirror/parser/event/EventStreamFileParser.java]: Done

File: src/main/java/com/hedera/mirror/parser/record/RecordFileLogger.java (hunk 3 above)
Comment (steven-sheehy) [reply src/main/java/com/hedera/mirror/parser/record/RecordFileLogger.java]: done

File: src/main/java/com/hedera/mirror/parser/record/RecordFileParser.java (hunk 4 above)
Comment (steven-sheehy) [reply src/main/java/com/hedera/mirror/parser/record/RecordFileParser.java]: Done

**Pull request merged**